dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/commands/PackageProcessor.java

```diff
@@ -479,6 +479,8 @@ private void addLibExtractDir(List<ArchiveEntryConfig> entryConfigs) throws IOEx
          * Add the archived loose files
          */
         getReferencedResources(entryConfigs);
+        for (ArchiveEntryConfig e : entryConfigs)
+            System.out.println(e);
```

Comment:
Assume this println was for debugging purposes. Should remove from the product code.